Assets/MixedRealityToolkit.Examples/Demos/ExamplesHub/Scripts/LoadContentScene.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit;
+#if SCENE_TRANSITIONS_ENABLED
+using Microsoft.MixedReality.Toolkit.Extensions.SceneTransitions;
+#endif
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+	public class LoadContentScene : MonoBehaviour
+	{
+		[SerializeField]
+		private LoadSceneMode loadSceneMode = LoadSceneMode.Single;
+        [SerializeField]
+        private string contentName = null;
```

Comment:
Could we change this type from a string to a SceneInfo object in case example scene names change we don't lose references?

Reply:
@Railboy could you help with this change?

Reply:
I can add this real quick then hook up the references!

Reply:
Just pushed up these changes: 
![image](https://user-images.githubusercontent.com/13305729/62254452-10db4e00-b3ae-11e9-9c6c-e465d71a6329.png)

Reply:
Thanks! Looking good!